src/Tribe/Ignored_Events.php

```diff
@@ -975,7 +977,35 @@ public function hook() {
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 
+			add_filter( 'wp_count_posts', [ $this, 'patch_count_posts' ], 10, 3 );
+
 			return true;
 		}
+
+		/**
+		 * Patch post-list-table queries to include ignored status,
+		 * as WP just accesses the stati without checking if they exist.
+		 *
+		 * @see wp-admin/includes/class-wp-posts-list-table.php->get_views()
+		 *
+		 * @since TBD
+		 *
+		 * @param object $counts       An object containing the current post_type's post
+		 *                              counts by status.
+		 * @param string $unused_type   Post type.
+		 * @param string $unused_perm   The permission to determine if the posts are 'readable'
+		 *                              by the current user.
+		 * @return object $counts       The modified object containing the current post_type's post
+		 *                              counts by status.
+		 */
+		public function patch_count_posts( $counts, $unused_type, $unused_perm ) {
+			$status = $this::$ignored_status;
+
+			if ( ! isset( $counts->$status ) ) {
```

Comment:
Would we want to check for an positive integer `>= 0` here; the input should not be trusted here.

Reply:
Why? This is just to check for situations when `$counts->$status` is not set and put in a placeholder value - I'm specifically using `isset()` so I don't change any existing values.

Reply:
Right, thanks.